### ISSUE TITLE
xDebug infinite page loading 

### ISSUE THREAD

**Issue by sawden** [dockware/dockware#5]
## How to reproduce the issue or behaviour (on Ubuntu 18.04).

1. Create a 'docker-compose.yml'
```
version: "3"

services:
        
    shopware:
      image: dockware/dev:latest
      container_name: shopware
      ports:
         - "80:80"
         - "3306:3306"
         - "22:22"
         - "8888:8888"
         - "9999:9999"
      networks:
         - web
      environment:
         - XDEBUG_ENABLED=1
         - XDEBUG_REMOTE_HOST=172.17.0.1

networks:
  web:
    external: false

```
2. Open terminal and start dockware with 'docker-compose up -d'
3. Download the source code.
```
mkdir -p ./src
docker cp shopware:/var/www/html/. ./src
```
4. Open PhpStorm and add the following settings:
**_SFTP_**
![sftp01](https://user-images.githubusercontent.com/36924392/85209770-c3a3fa00-b33a-11ea-8bfa-aa069227ce16.png)
![sftp02](https://user-images.githubusercontent.com/36924392/85209773-c6065400-b33a-11ea-8d6d-18d46560b425.png)
![sftp03](https://user-images.githubusercontent.com/36924392/85209775-c7d01780-b33a-11ea-9a74-2d209da8e818.png)
_**xDebug**_
![xdebug01](https://user-images.githubusercontent.com/36924392/85209824-4036d880-b33b-11ea-9615-c87dbd35a364.png)
_**Servers**_
![servers01](https://user-images.githubusercontent.com/36924392/85209854-88ee9180-b33b-11ea-99dd-1c20b7d34af7.png)
5. Install 'Xdebug helper' for Chrome and set the IDE in the settings to:
![XDB01](https://user-images.githubusercontent.com/36924392/85209910-05817000-b33c-11ea-8b88-80d02a4c408e.png)
6. Switch to PhpStorm and set a breakpoint in src/public/index.php
![br](https://user-images.githubusercontent.com/36924392/85210124-93118f80-b33d-11ea-9bbd-3286c872ef78.png)
7. 'Start Listening for PHP Debug Connections' by clicking the icon in the right top corner
![clicktc](https://user-images.githubusercontent.com/36924392/85210012-e33c2200-b33c-11ea-95d0-dbf5186504a8.png)
8. Open http://localhost/ in your Chrome browser and enable XDebug by clicking on the option "Debug"
9. The Page is loading infinite and PhpStorm is not responding to the breakpoint...

Did I forget something?
Hope someone can help here 💙💜

**Comment by boxblinkracer** [comment]
hi
sorry to hear that its not working
could you contact me in the slack community workspace? 
thx 
Christian Dangl

**Comment by sawden** [comment]
Fixed by updating PhpStorm 2018.1.7 to 2020.1.2 🙂👍

**Comment by boxblinkracer** [comment]
amazing, so it was really that weird problem
thanks for letting us know :)
